packages/terra-date-time-picker/tests/nightwatch/DateTimePickerTests.jsx

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable import/no-extraneous-dependencies */
+
+import React from 'react';
+import { Link } from 'react-router';
+
+const DateTimePickerTests = () => (
+  <div>
+    <ul>
+      <li><Link to="/tests/date-time-picker-tests/default">DateTimePicker - Default</Link></li>
+      <li><Link to="/tests/date-time-picker-tests/date-only">DateTimePicker - Date Only</Link></li>
+      <li><Link to="/tests/date-time-picker-tests/date-time">DateTimePicker - Date Time</Link></li>
+      <li><Link to="/tests/date-time-picker-tests/dst">DateTimePicker - DST</Link></li>
+      <li><Link to="/tests/date-time-picker-tests/exclude-dates">DateTimePicker - Exclude Dates</Link></li>
+      <li><Link to="/tests/date-time-picker-tests/filter-dates">DateTimePicker - Filter Dates</Link></li>
+      <li><Link to="/tests/date-time-picker-tests/include-dates">DateTimePicker - Include Dates</Link></li>
+      <li><Link to="/tests/date-time-picker-tests/min-max">DateTimePicker - Min Max</Link></li>
```

Comment:
By Min Max do you mean date range?

Reply:
This is for setting a minimum and maximum date that are allowed to be selected and no so much a date range. I look at date range as selecting two separate dates to indicate a from date and a to date.